lib/community/solicitor.js

```diff
@@ -46,13 +57,17 @@ var queueEmail = function (emailData) {
 }
 
 Solicitor.sendToUser = function (user, community) {
-  return queueEmail(emailData(user, community))
+  if (user.get('settings').email_solicitation_preference) {
+    return queueEmail(emailData(user, community))
+  } else {
+    return Promise.resolve()
+  }
```

Comment:
It's fine, just FYI. Would be a bit more readable to use Promise.method in cases with more branching.